assets/sass/v2/_select-authenticator.scss

```diff
@@ -52,6 +52,10 @@
     .authenticator-label {
       width: 175px;
     }
+
+    &--with-sublabel .authenticator-label {
+      line-height: 1;
```

Comment:
Note: I'm talking to @vincentvoong-okta about this change. Will only merge if UX is ok with how it looks spacing-wise on the screenshot.